vcztools/cli.py

```diff
@@ -53,6 +53,13 @@ def index(path):
     default=None,
     help="Samples to include.",
 )
+@click.option(
+    "-G",
+    "--drop_genotypes",
```

Comment:
```suggestion
    "--drop-genotypes",
```

Nit: I think Click can use either underscore or hyphen, but this is consistent with the other commands.